lib/WorldState.js

```diff
@@ -79,11 +78,12 @@ class WorldState {
    * @param {string} json The worldstate JSON string
    * @param {Object} [deps] The options object
    */
-  constructor(json, deps = defaultDeps) {
+  constructor(json, lang = 'en_US', deps = defaultDeps) {
```

Comment:
Ok but how do you represent that in Alert class for example ? Because `alerts[0]` is an `Alert` object but `alerts[0].translations['fr_FR']` too... And a Alert class that contains Alert... can be deep if we didn't rework that.